### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Innocent259**
In this project milestone I:

1. Implemented the views according to the given wireframe
2. Added basic styling using CSS

### CODE REVIEW

File: app/views/users/index.html.erb
Comment (KanzaTahreem) [inline app/views/users/index.html.erb:1]: - [ ] You've done an excellent job so far 👏✨ We're almost there. Please ensure that the index view page aligns with the provided [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png).  You can add your personal touch to the styling, but let's make sure the overall structure and elements align closely with the given design 🎨


Current UI | Wireframe
---| ---|
image | image

File: app/views/users/show.html.erb
Comment (KanzaTahreem) [inline app/views/users/show.html.erb:1]: - [ ] Here as well, please ensure that the index view page aligns with the provided [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAAB5V373VFXXMF4ELUDKAJMRKZLUI56Q). You can add your personal touch to the styling, but let's make sure the overall structure and elements align closely with the given design 🎨

Current UI | Wireframe
---| ---|
image | image

File: app/views/posts/index.html.erb
Comment (KanzaTahreem) [inline app/views/posts/index.html.erb:1]: - [ ] Similarly here, please ensure that the index view page aligns with the provided [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png). You can add your personal touch to the styling, but let's make sure the overall structure and elements align closely with the given design 🎨

Current UI | Wireframe
---| ---|
image | image

File: app/views/posts/show.html.erb
Comment (KanzaTahreem) [inline app/views/posts/show.html.erb:1]: - [ ] Similarly as above, please ensure that the index view page aligns with the provided [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAAB5V373VPZ32HAADXEVBSMDOZLUJIKQ). You can add your personal touch to the styling, but let's make sure the overall structure and elements align closely with the given design 🎨

Current UI | Wireframe
---| ---|
image | image

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -1,15 +1,54 @@
...
+
+.user-container {
+  display: flex;
+  gap: 2rem;
+}
+
+.user-img {
+  border: 0.5px solid #131212;
+}
+
```
Comment (KanzaTahreem) [inline app/assets/stylesheets/application.css]: - [ ] Fantastic progress so far! 💪✨ Please ensure the image styling aligns precisely with the wireframe guidelines Currently, when I'm using a valid image URL, this is how the image is being displayed 👇

image

File: app/views/users/_user.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<% @users.each do |user| %>
+    <div class="user-container">
+        <img class="user-img" src="<%= user.photo %>" alt="User photo">
+        <div class="user-post">
+          <a class="username" href="<%= user_path(user.id) %>"><%= user.name %></a>
+          <p class="user-posts">Number of posts: <%= user.posts.count %></p>
+        </div>  
+    </div>
+<% end %>
```
Comment (KanzaTahreem) [inline app/views/users/_user.html.erb:9]: _Well done for using partial 👏💯💯_

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @Innocent259 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Methods present in the models are used
- ✅ Descriptive PR and README file
- ✅ Linters are working properly

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Innocent259 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- All views are now following given wireframes :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**